scrapers/AdultEmpireCash.yml

```diff
@@ -0,0 +1,126 @@
+      - mypervyfamily.com/
+      - pornstarstroker.com/ # aggregator site?
+      - reaganfoxx.com/
+      - severesexfilms.com/
+      - thirdworldxxx.com/
+      - touchmywife.com/
+      - westcoastproductions.com/
+
+  # VR Sites
+  - action: scrapeXPath
+    scraper: vrSceneScraper
+    url:
+      - lethalhardcorevr.com/
+      - whorecraftvr.com/
+
+movieByURL:
+  - action: scrapeXPath
+    scraper: movieScraper
+    url:
+      # - bangmytransass.com/ # Scenes published as movies?
+      # - blacktranswhackers.com/ # Scenes published as movies?
+      - concoxxxion.com/
+      - elegantangel.com/
+      - manipulativemedia.com/
+      - severesexfilms.com/
+      # - shemalestrokers.com/ # Scenes published as movies?
+      - thirdworldxxx.com/
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h1[@class="description"]/text()
+      Date:
+        selector: //div[@class="release-date"][1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Details: //div[@class="synopsis"]//text()
+      Tags:
+        Name: //div[@class="tags"]/a/text()
+      Performers:
+        Name: //div[@class="video-performer"]/a//text()
+      Studio:
+        Name:
+          selector: //div[@class="studio"]/span[2]
+          postProcess:
+            # Override studio name only when needed
+            - map:
+                Jay Rock Clips: Jay's POV
+            - replace:
+                # 'Jane Doe Clips' => 'Jane Doe'
+                - regex: \sClips$
+                  with:
+      Image: //link[@rel="image_src"]/@href
+      Movies:
+        Name: //h1[@class="description"]/following-sibling::p/a/text()
+
+  movieScraper:
+    movie:
+      Name: //h1[@class="description"]/text()
+      Date:
+        selector: //div[@class="release-date"][1]/text()
+        postProcess:
+          - parseDate: Jan 02, 2006
+      Synopsis: //div[@class="synopsis"]//text()
+      Director: //div[@class="director"]/a/text()
+      Duration:
+        selector: //div[@class="release-date"][2]/text()
+        postProcess:
+          - replace:
+            - regex: " hrs. "
+              with: ":"
+            - regex: " mins."
+              with: ":00"
+      Studio:
+        Name:
+          selector: //div[@class="studio"]/a
+          postProcess:
+            - replace:
+                # 'Jane Doe Clips' => 'Jane Doe'
+                - regex: \sClips$
+                  with:
+      FrontImage: //link[@rel="image_src"]/@href
+
+  vrSceneScraper:
+    common:
+      $content: //div[@class="item-page-details"]
+    scene:
+      Title: $content//h1
+      Details:
+        selector: $content//div[@id="synopsis-full"]
+        concat: "\n\n"
+      Performers:
+        Name:
+          selector: $content//span[@class="overlay-inner"]//small
+          postProcess:
+            - replace:
+                - regex: \((.+)\)
+                  with: $1
+      Image:
+        selector: //link[@rel="image_src"]/@href'
```

Comment:
the image selector doesnt seem to work for a couple of scenes i tried for both vr sites ( `https://lethalhardcorevr.com/3024943/secret-sex-streaming-vr-oculus-porn-videos.html` , `https://whorecraftvr.com/1972917/mount-me-for-a-mount-streaming-vr-oculus-porn-videos.html`  , `https://whorecraftvr.com/1972917/mount-me-for-a-mount-streaming-vr-oculus-porn-videos.html` )
it seems that the image needed is in a style block as a background

Reply:
Thank you very much!
It was the stray single quote at the end of the selector, but mostly it was the `$1_lg` replacement, which should have been `${1}_lg`.
Confirmed the corrected one works in Stash.